examples/babel-webpack-4/serverless.yml

```diff
@@ -2,6 +2,7 @@ service: babel-webpack-4-example
 
 # Add the serverless-webpack plugin
 plugins:
+  - serverless-offline
```

Comment:
I think this should be reversed, shouldn't it? The sls-webpack hooks should precede the offline hooks internally.

Reply:
It worked for me this way, but I realized that at `README` we state to use `serverless-offline` after `serverless-webpack` so I'm going to change it.